### PR TITLE
Remove some links still pointing to wiki.egi.eu

### PR DESCRIPTION
**Pull request by gwarf**
Remove some links still pointing to wiki.egi.eu.
Some are still to be updated.

### CODE REVIEW

File: content/en/providers/getting-started/integration_checklist/_index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Resource Centre integrated into the EGI Federation.
 
 1. Security:
 
-   - HTC: [pakiti](https://wiki.egi.eu/wiki/EGI_CSIRT:Pakiti_client) is
+   - HTC: [pakiti](../../../internal/security-coordination/monitoring/pakiti) is
      installed and the outcome of the EGI CSIRT assessment is positive;
    - Cloud: the
      [EGI security Survey](https://www.surveymonkey.com/r/Cloud_Security_Questionnaire_for_Resource_Centres)
```
Comment (enolfc) [inline content/en/providers/getting-started/integration_checklist/_index.md:41]: Just noticed this link is also gone. I don't recall the latest URL

File: content/en/users/data/storage/block-storage/_index.md [hunk 2]
```diff
@@ -64,11 +64,6 @@ of a provider, from a [command-line interface](#manage-from-the-command-line)
 (CLI), or via the
 [OpenStack Block Storage API](https://docs.openstack.org/api-ref/block-storage/).
 
-{{% alert title="Note" color="info" %}} Please refer to
```
Comment (enolfc) [inline content/en/users/data/storage/block-storage/_index.md:67]: wow, occi still around after years without it anywhere

**Review by enolfc — COMMENTED** [review]
Some minor comments

File: content/en/providers/getting-started/integration_checklist/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/providers/getting-started/integration_checklist/_index.md:41]: I've been already asking Ale about the new survey url, we will see what we can find.

**Comment by gwarf** [comment]
The markdown link check error with https://opensource.org/licenses is a false negative, it can be ignored.

File: content/en/providers/getting-started/integration_checklist/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/providers/getting-started/integration_checklist/_index.md]: OK, so in fact the survey got migrated last year, this reference just got forgotten.
The new link was documented in https://go.egi.eu/sec05

**Review by paolini78 — APPROVED** [review]
thanks

**Comment by gwarf** [comment]
@enolfc what should we do for the (broken) link to the "Federated Cloud Architecture" shown at https://docs.egi.eu/providers/cloud-compute/requirements/ ? 
There is also a link to https://wiki.egi.eu/wiki/Federated_Cloud_infrastructure_status on https://docs.egi.eu/providers/cloud-compute/validation/ . Content on this wiki page seems very outdated.

**Comment by enolfc** [comment]
> @enolfc what should we do for the (broken) link to the "Federated Cloud Architecture" shown at https://docs.egi.eu/providers/cloud-compute/requirements/ ? 

https://docs.egi.eu/users/getting-started/architecture/ should work here

> There is also a link to https://wiki.egi.eu/wiki/Federated_Cloud_infrastructure_status on https://docs.egi.eu/providers/cloud-compute/validation/ .  Content on this wiki page seems very outdated.

This can be just removed.

> In howto04, there is also some reference to https://wiki.egi.eu/wiki/Federated_Cloud_APIs_and_SDKs, a broken link too.

this one is so outdated that it hurts. I think it's better to remove the whole cloud section and provide new documentation in a subsequent PR than trying to fix few links here.

**Pull request merged**